.travis.yml

```diff
@@ -28,6 +29,12 @@ matrix:
     gemfile: gemfiles/rails_5.0.gemfile
   - rvm: 2.1.10
     gemfile: gemfiles/rails_5.0.gemfile
+  - rvm: 1.9.3
+    gemfile: gemfiles/rails_5.1.gemfile
+  - rvm: 2.0.0
+    gemfile: gemfiles/rails_5.1.gemfile
+  - rvm: 2.1.10
+    gemfile: gemfiles/rails_5.1.gemfile
```

Comment:
Let's swap this to an `include:` for 4.2 + old rubies, so the list doesn't keep expanding